release/release_gold.sh

```diff
@@ -1,9 +1,23 @@
 #!/bin/bash
 
 source _liferay_common.sh
+file="release-data/release.properties"
+
+function add_tag_to_release {
+    curl -L \
+        -X POST \
+        -H "Accept: application/vnd.github+json" \
+        -H "Authorization: Bearer ${GIT_PAT}" \
+        -H "X-GitHub-Api-Version: 2022-11-28" \
+        https://api.github.com/repos/"${GIT_REPO_OWNER}"/liferay-portal-ee/git/tags \
+        -d '{"tag":"'"${LIFERAY_RELEASE_VERSION}"'",
+            "message":"initial version",
+            "object":"'"$(get_release_properties_value git.hash.liferay-portal-ee)"'",
+            "type":"commit"}'
+}
 
 function check_usage {
-	if [ -z "${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" ] || [ -z "${LIFERAY_RELEASE_VERSION}" ]
+	if [ -z "${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" ] || [ -z "${LIFERAY_RELEASE_VERSION}" ] || [ -z "${GIT_REPO_OWNER}" ] || [ -z "${GIT_PAT}" ]
```

Comment:
GIT_REPO_OWNER should not ever be empty, so no check is required.
It should be defined with default value, though.

Reply:
For default should we add GITHUB_REPOSITORY_OWNER:=liferay?
As for the GITHUB_PAT, what should be the default value?